src/webgpu/web_platform/copyToTexture/canvas.spec.ts

```diff
@@ -625,6 +625,12 @@ g.test('copy_contents_from_gpu_context_canvas')
       .combine('width', [1, 2, 4, 15, 255, 256])
       .combine('height', [1, 2, 4, 15, 255, 256])
   )
+  .before(async t => {
+    const { srcAndDstInSameGPUDevice } = t.params;
+    if (!srcAndDstInSameGPUDevice) {
```

Comment:
like the device_mismatch tests, I think we can remove the conditional here to be simpler

Reply:
done